packages/react-integration/demo-app-ts/src/components/demos/TopologyDemo/TopologyDemo.tsx

```diff
@@ -274,6 +276,13 @@ export const TopologyDemo: React.FunctionComponent = () => {
             </Tab>
           </Tabs>
         </Tab>
+        <Tab eventKey={TOPOLOGY_BASELINE} title={<TabTitleText>Baseline demo</TabTitleText>}>
```

Comment:
Do you think it would be valuable to showcase an out-of-the-box working demo? Would it be helpful to include a "Baseline demo" in the demo app like this?

Reply:
Yes, I think it's good to have a simple demo of the complete setup that applications would typically use. Unsure if I think we should include the options here or just use some simple Component that better shows how to go from backend data to a model to displaying the component.